test/powershell/Modules/Microsoft.PowerShell.Management/Get-Location.Tests.ps1

```diff
@@ -2,20 +2,76 @@
 # Licensed under the MIT License.
 Describe "Get-Location" -Tags "CI" {
     $currentDirectory=[System.IO.Directory]::GetCurrentDirectory()
+
     BeforeEach {
-	Push-Location $currentDirectory
+        Push-Location $currentDirectory
     }
 
     AfterEach {
-	Pop-location
+        Pop-location
```

Comment:
Please put line 4 to BeforeAll block.

It seems BeforeEach and AfterEach is used only in one test and we could move the code to the test.